src/test/java/com/epam/tc/hw2/exercise2/SiteSelectorsTests.java

```diff
@@ -0,0 +1,288 @@
+package com.epam.tc.hw2.exercise2;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import java.util.concurrent.TimeUnit;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchSessionException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.Select;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class SiteSelectorsTests {
+    WebDriver webDriver;
+
+    @BeforeClass
+    public void setupClass() {
+        WebDriverManager.chromedriver().setup();
+    }
+
+    @BeforeMethod
+    public void setupTest() {
+        webDriver = new ChromeDriver();
+        webDriver.manage().window().maximize();
+        webDriver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
+    }
+
+    @AfterMethod
+    public void teardown() {
+        if (webDriver != null) {
+            webDriver.quit();
+        }
+    }
```

Comment:
Could be extracted to the base test class

Reply:
fixed